Deploy/stacks/dynamic/stack-manager/README-podman.md

```diff
@@ -0,0 +1,57 @@
+# Running the stack with podman
+
+It is assumed that the reader is familiar with [how to operate the stack using Docker](./README.md). Using [podman](https://podman.io/) is largely similar, and this readme addresses only points where differences arise.
+
+## Installing podman
+
```

Comment:
Is it worth mentioning which versions of Podman are expected to work as expected?
I tried the versions shipped with some of the last few Ubuntus and only the latest, with podman version 4.9.3, worked.

Reply:
Have you tried 4.6.1?

Reply:
Addressed in commit ad1dd54e6e9e692f7f740b70c1e0dcbee8267f49.